### ISSUE TITLE
Accessibility testing flags errors due to missing labels

### ISSUE THREAD

**Issue by lucywoodman** [lucywoodman/the-chocolate-factory#3]
# Profile/Checkout: Accessibility testing flags errors due to missing labels

The fields are flagged on Wave accessibility test for the Profile and Checkout forms. The labels were removed to keep the styling of the form in sync with Stripe's payment field.

A solution is to allow the labels to display, then use Crispy Form tags instead of filters to take advantage of the Layout classes. Then add the `sr-only` class to the labels. However, this breaks form submission, which needs further investigation.
